src/biokbase/narrative/tests/test_jobcomm.py

```diff
@@ -423,6 +423,42 @@ def test_start_job_status_loop__cell_ids(self):
                 self.assertFalse(self.jc._running_lookup_loop)
                 self.assertIsNone(self.jc._lookup_timer)
 
+    @mock.patch(CLIENTS, get_failing_mock_client)
+    def test_start_job_status_loop__initialise_jobs_error(self):
+        # check_workspace_jobs throws an EEServerError
+        self.jc.start_job_status_loop(init_jobs=True)
+        self.assertEqual(
+            self.jc._comm.last_message,
+            {
+                "msg_type": ERROR,
+                "content": {
+                    "error": "Unable to get initial jobs list",
+                    "message": "check_workspace_jobs failed",
+                    "code": -32000,
+                    "source": "ee2",
+                    "name": "JSONRPCError",
+                },
+            },
+        )
+        self.assertFalse(self.jc._running_lookup_loop)
+
+    @mock.patch(CLIENTS, get_mock_client)
+    def test_start_job_status_loop__no_jobs_stop_loop(self):
```

Comment:
Similarly, test that trying to run the lookup loop with 0 registered jobs is unsuccessful.